Automattic/wp-calypso
# Update jetpack masterbar design

## Proposed Changes

* Update masterbar design to match the new comps
* Fix issue where the "backdrop" was able to be scrolled past resulting in this
![image](https://github.com/Automattic/wp-calypso/assets/65001528/60cd9eae-6220-4cf6-bc7a-ae898fcd76b3)
* Fix issue where logo and gravatar was not showing when products submenu was open (it is on jetpack.com)
* Makes header sticky like on jetpack.com instead of these things
![image](https://github.com/Automattic/wp-calypso/assets/65001528/0557f695-8d50-4a63-bfef-acda44d402cd)
* Make sure the user menu (when logged in) still works as expected
* Make sure the mobile menu works as expected.
* Try opening in incognito initially and making sure both desktop and mobile still work
* Make sure clicking on the blur background closes the megamenu
* Log out and make sure all menus still work as expected

Figma: 22h2u6LRIyamGpA0vEo6hW-fi-3708%3A657
P2: p1HpG7-nDJ-p2

## Testing Instructions

1. Click on the Calypso Green live link or checkout this branch and start your local environment via `yarn start-jetpack-cloud`
2. Go to `/pricing`
3. Make sure the header looks and functions exactly the same on mobile and desktop as it does in this diff: D118381-code as well as looks the same as in the Figma comps
4. Make sure none of the fixes listed in the Proposed changes broke anything else on the header or caused any unexpected issues

## Pre-merge Checklist
